### ISSUE TITLE
Long Content Scrolling modal dismiss when drag the scroll bar eventhough on valor demo

### ISSUE THREAD

**Issue by thanhdn1989** [valor-software/ng2-bootstrap#5763]
Bug description:
When using the Long Content Scrolling modal, it is auto dismiss when i scroll by dragging the scroll bar. The demo on official site have the same problem

Versions of ngx-bootstrap, Angular, and Bootstrap:
ngx-bootstrap:
5.5.0
Angular:
9
Bootstrap:
4.4.1
Build system: Angular CLI, System.js, webpack, starter seed:
Angular CLI
**Expected behavior**
Modal not dismiss when i scroll by dragging the scroll bar

**Comment by MarkoSulamagi** [comment]
I can confirm that this is an issue. I also tested it on official demo site and the same problem.
I had to temporarily set `ignoreBackdropClick: true,` .

**Comment by thanhdn1989** [comment]
> I can confirm that this is an issue. I also tested it on official demo site and the same problem.
> I had to temporarily set `ignoreBackdropClick: true,` .

yes, but the sad is that my client did not accept this workaround

**Comment by MarkoSulamagi** [comment]
@thanhdn1989, eventually we decided to pin ngx-bootstrap version to 5.4.0. It works with that version still. 
I think it also works with a bit newer version, but we decided on 5.4.0 because that solved another problem we had with dropdown as well.
